src/ai/backend/manager/api/session.py

```diff
@@ -2101,17 +2104,23 @@ async def init(app: web.Application) -> None:
     # Scan ALIVE agents
     app_ctx.agent_lost_checker = aiotools.create_timer(
         functools.partial(check_agent_lost, root_ctx), 1.0)
-    app_ctx.stats_task = asyncio.create_task(stats_report_timer(root_ctx))
+    app_ctx.stats_task = aiotools.create_timer(
+        functools.partial(report_stats, root_ctx), 1.0,
```

Comment:
Previously it used `asyncio.sleep(5)`. Why is the interval is changed to 1.0?

Reply:
https://github.com/lablup/backend.ai-manager/pull/533/commits/f2a76811644790d5195c8a8fc814d9bf1cb6905f resolves this issue.